<html>
When an exception occurs, it's useful to include the entire traceback in your log file for diagnosing problems.
<br/>
<br/>
You can use the "exception()" function on the Logger instance to gather the traceback information and do the
formatting for you. It's similar to the "debug()" function, except you should only call "exception()" from an
exception handler.
<br/>
<br/>
The following code generates an exception. Replace the call to "pass" with a call to log the exception instead. This
will also allow the exception to propagate up to the caller. This is usually better in this case, instead of silently
catching any exception.
</html>